clustered/server/src/main/java/org/ehcache/clustered/server/ClusterTierManagerPassiveEntity.java

```diff
@@ -42,34 +42,7 @@
   private final EhcacheStateService ehcacheStateService;
   private final Management management;
 
-  @Override
-  public void invoke(EhcacheEntityMessage message) {
-
-    try {
-      if (message instanceof EhcacheOperationMessage) {
-        EhcacheOperationMessage operationMessage = (EhcacheOperationMessage) message;
-        EhcacheMessageType messageType = operationMessage.getMessageType();
-        if (isLifecycleMessage(messageType)) {
-          invokeLifeCycleOperation((LifecycleMessage) message);
-        } else if (isPassiveReplicationMessage(messageType)) {
-          try {
-            invokeRetirementMessages((PassiveReplicationMessage)message);
-          } catch (ClusterException e) {
-            LOGGER.error("Unexpected exception raised during operation: " + message, e);
-          }
-        } else {
-          throw new AssertionError("Unsupported EhcacheOperationMessage: " + operationMessage.getMessageType());
-        }
-      } else {
-        throw new AssertionError("Unsupported EhcacheEntityMessage: " + message.getClass());
-      }
-    } catch (ClusterException e) {
-      // Reaching here means a lifecycle or sync operation failed
-      throw new IllegalStateException("A lifecycle or sync operation failed", e);
-    }
-  }
-
-  public ClusterTierManagerPassiveEntity(ClusterTierManagerConfiguration config,
+  ClusterTierManagerPassiveEntity(ClusterTierManagerConfiguration config,
```

Comment:
Not really something under the scope of this issue. The same is applicable for other entity classes as well. IDE prompted and since I was touching this file anyway I did it. Want me to revert?